internal/pkg/manifest/testdata/backend-svc-customhealthcheck.yml

```diff
@@ -37,4 +37,6 @@ exec: true     # Enable running commands in your container.
 # You can override any of the values defined above by environment.
 #environments:
 #  test:
-#    count: 2               # Number of tasks to run for the "test" environment.
\ No newline at end of file
+#    count: 2               # Number of tasks to run for the "test" environment.
+#    deployment:            # deployment strategy for the "test" environment.
+#       rolling: 'recreate' # key is rolling deployment and value is deployment strategy name i.e. either default or recreate.
```

Comment:
i'm wondering if we think `deployment` is common enough that it's worth putting into the manifest template? i don't want to overwhelm with _all_ of the options in the template. any thoughts on this?

Reply:
So do you mean we shouldn't put it in all the templates and keep it few or you suggest not to keep it at all in the manifest templates?

Reply:
basically i'm thinking about removing these lines from all of the manifest templates
```
#    deployment:            # deployment strategy for the "test" environment.
#       rolling: 'recreate' # key is rolling deployment and value is deployment strategy name i.e. either default or recreate.
```

the idea being we'd document the fields in our documentation, but not necessarily include them in the default template. i'm not really decided one way or the other tbh, so i'm fine leaving it as you have it for now.

Reply:
I feel like it'd be a nice little hint for users to be aware that they could use "recreate" for their test environment. The benefit would be a faster deployment in their test deployment. I'm fine with either though!

Reply:
Makes sense, that does sound like a common want/pretty nice benefit. Let's leave it in the generated manifest files!